ulab_Crunch_Numbers_Fast/benchmark.py

```diff
@@ -1,11 +1,21 @@
-import time
+import utime
 import math
 import ulab
 import ulab.numerical
 
+def timeit(f, *args, **kwargs):
+    func_name = str(f).split(' ')[1]
+    def new_func(*args, **kwargs):
+        t = utime.ticks_us()
+        result = f(*args, **kwargs)
+        print('execution time: ', utime.ticks_diff(utime.ticks_us(), t), ' us')
```

Comment:
ticks_diff and ticks_us do not exist in CircuitPython.  We have time.monotonic_ns on all our platforms.  Naturally you can divide by 1000 to convert to us.

Reply:
OK.